### PR TITLE
group spirits by expansion and show their full names in spirit dropdown

### PR DESCRIPTION
**Pull request by petertseng**
closes https://github.com/nathanj/spirit-island-pbp/issues/60 if the Apocrypha category and Exploratory Testing category are sufficiently clear that these spirits are set apart from the rest.

---

It can be a little cryptic to only show one word of the name, and for
some spirits it's not clear which word I should look for

For example:

* Fractured Days Split the Sky: Fractured or Days?
* Shifting Memory of Ages: Shifting or Memory?
* Shroud of Silent Mist: Mist or Shroud?

By showing the full name, there is no longer any question.
Disadvantage: the dropdown does get quite wide.

### CODE REVIEW

**Comment by samsullivan** [comment]
I think this looks great despite the larger size, probably a net benefit; doesn't look any more horrible on mobile.

**Comment by samsullivan** [comment]
Will approve when I have a chance to spot check all the names for typos :joy:

File: pbf/views.py [hunk 1]
```diff
@@ -345,42 +345,92 @@ def view_game(request, game_id):
...
+        # Don't think so; not everyone knows the complexities by heart,
+        # and so the ordering would look haphazard to those who don't know them.
+        #
+        # Similarly, aspects are going to be grouped with their spirit
+        # regardless of which expansion the aspect was released in,
+        # because not everyone is going to know which expansion has which aspects.
+        'Spirit Island (base)': [
+            ('Bringer', 'Bringer of Dreams and Nightmares', ('Enticing', 'Violence')),
+            ('Lightning', "Lightning's Swift Strike", ('Immense', 'Pandemonium', 'Sparking', 'Wind')),
+            ('Ocean', "Ocean's Hungry Graps", ('Deeps',)),
```
Comment (petertseng) [inline pbf/views.py:362]: Graps? what is a Grap?

**Review by petertseng — COMMENTED** [review]
funny you should mention typos...

File: pbf/views.py (hunk 1 above)
Comment (samsullivan) [reply pbf/views.py]: There's always bound to be one when writing this much 😂

**Comment by TheOriginalSoni** [comment]
`Covets Gleaming Shards of Earth [Apocrypha]` and `Spreading Rot Renews the Earth [Apocrypha]` please.

That is a stronger identifier than just reading the name when scrolling.

Also, quick check, this does not change the spirit name right? Just in the selection drop down? Because the tabs for players + spirits takes up valuable real estate as it is

**Comment by petertseng** [comment]
No change to the spirit name display in this pull request. That would require changing tabs.html and this pull request didn't touch that file.

**Comment by petertseng** [comment]
So to confirm, Apocrypha category would now look like this?

apocrypha

**Review by samsullivan — APPROVED** [review]

**Pull request merged**